static/src/javascripts/projects/commercial/modules/mobile-sticky.js

```diff
@@ -30,7 +33,10 @@ const createAdWrapper = () => {
 };
 
 export const init = (): Promise<void> => {
-    if (shouldIncludeMobileSticky()) {
+    if (
+        shouldIncludeMobileSticky() ||
+        isInABTestSynchronous(commercialUkRowMobileSticky)
```

Comment:
Why not have this check inside the `shouldIncludeMobileSticky()` function?

Reply:
Done as suggested